content/en/development/design-docs/blb-migration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Basic to Standard Load Balancer Migration"
+linkTitle: "Basic to Standard Load Balancer Migration"
+type: docs
+description: >
+    Support migration from Basic Load Balancer to Standard Load Balancer.
+---
+
+## Background
+
+On September 30, 2025, Basic Load Balancer will be [retired](https://learn.microsoft.com/en-us/azure/load-balancer/skus). This design document describes the planned migration path from Basic Load Balancer to Standard Load Balancer.
+
+## Design
+
+> The following `User` stands for human users, or any cluster provisioning tools (e.g., AKS).
```

Comment:
If SLB outbound not working, could we migrate just after user change sku to standard and restart ccm?